source/PCA9685Support.cpp

```diff
@@ -383,13 +386,25 @@ HRESULT PCA9685Device::_InitializeChip(ULONG i2cAdr)
         // Queue a read of the chip MODE1 register to get the state of the SLEEP bit.
         //
 
+
         I2cTransactionClass transaction;
         UCHAR readBuf[1] = { 0 };                   // Buffer for reading data from chip
         UCHAR mode1RegAdr[1] = { MODE1_ADR };       // Buffer for MODE1 register address
         UCHAR mode2RegAdr[1] = { MODE2_ADR };       // Buffer for MODE2 register address
         UCHAR preScaleAdr[1] = { PRE_SCALE_ADR };   // Buffer for PRE_SCALE register address
-        MODE1 mode1Reg = { 0, 0, 0, 0, 0, 1, 0, 0 }; // No sleep, auto-increment, internal clock
-        MODE2 mode2Reg = { 0, 1, 1, 0, 0 };         // Drive outputs both high & low, change on ACK, non-inverted
+        MODE1 mode1Reg	     = { 0, 0, 0, 0, 0, 1, 0, 0 }; // No sleep, auto-increment, internal clock
+        MODE1 mode1RegSleep  = { 0, 0, 0, 1, 0, 1, 0, 0 }; // Sleep, auto-increment, internal clock
+        MODE2 mode2Reg       = { 0, 1, 1, 0, 0 };         // Drive outputs both high & low, change on ACK, non-inverted
+
+        // Software Reset
+        // 
+        hr = transaction.setAddress(0);
```

Comment:
It is correct that using I2C address is mentioned in the chip datasheet, and it may very well be that the BusProvider does this--however, it is a dangerous thing to do from a reliability and compatibility point of view.  We are writing code for a chip that sits on an I2C bus that is not dedicated exclusively to this chip or a fixed set of chips that we control.  End users can, and will, add chips to the I2C bus and expect them to function correctly.  By sending data to address 0x00 on the bus you are sending it to any attached devices that support the general call address.  The general call address is designed as an efficient way to access multiple chips in a locked down, fixed environment.